test/tpu/run_pallas_test.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+set -xue
+CDIR="$(cd "$(dirname "$0")" ; pwd -P)"
```

Comment:
Please follow the convention of starting an internal env var with a `_`. See `run_tests.sh` for examples.